riotfile.py

```diff
@@ -337,7 +337,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="cherrypy",
-            command="pytest {cmdargs} tests/contrib/cherrypy",
+            command="python -m pytest {cmdargs} tests/contrib/cherrypy",
```

Comment:
is this still required?

Reply:
Ah! Apparently they are no longer needed! Thanks! 😃